test/spec/auctionmanager_spec.js

```diff
@@ -232,21 +232,23 @@ describe('auctionmanager.js', function () {
       assert.deepEqual(response, expected);
     });
 
-    it('No bidder level configuration defined - default for video', function () {
-      config.setConfig({
-        cache: {
-          url: 'https://prebid.adnxs.com/pbc/v1/cache'
-        }
+    if (FEATURES.VIDEO) {
```

Comment:
I am wondering, when the unit test is run, if we want to test the video feature, do we have to set the FEATURES.VIDEO flag to true in order to test it?  Does prebid have test on different flag type?

Reply:
Great question. Running [`npm test`](https://github.com/WPMedia/Prebid.js/blob/master/package.json#L7) actually runs the [`test` Gulp task](https://github.com/WPMedia/Prebid.js/blob/master/gulpfile.js#L453) which executes the unit tests in two phases: the first with all features _disabled_ and the second with everything enabled. We don't have any CI enabled in our fork but I'm running `npm test` with every file I modify to ensure everything stays nice and 🟢